cli/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -532,11 +532,15 @@ pub fn global_manifest_lockfile_path(flox: &Flox) -> PathBuf {
     path
 }
 
+/// Stores a found DotFlox and whether it was found in the starting directory
+pub type FoundDotFlox = (DotFlox, bool);
```

Comment:
I'm not sure I like this API change. Yes, it makes these couple of tests easier to write, but you know have data that, as far as I can tell, is only useful for testing purposes leaking into the normal usage of `find_dot_flox`. See `EnvironmentSelect`, `detect_environment`, etc where you now have a bunch of `(found, _)` instances.

It should still be possible to write the test without this API change, right?

Reply:
Yeah it's cleaner to just drop it, fixed